shadows/framework/src/main/java/org/robolectric/shadows/ShadowInstrumentation.java

```diff
@@ -772,7 +772,7 @@ protected void unbindService(final ServiceConnection serviceConnection) {
             Logger.warn(
                 "Configured to call onServiceDisconnected when unbindService is called. This is"
                     + " not accurate Android behavior. Please update your tests and call"
-                    + " ShadowActivity#setUnbindCallsOnServiceDisconnected(false). This will"
+                    + " ShadowActivity#setUnbindServiceCallsOnServiceDisconnected(false). This will"
```

Comment:
The method is not even on `ShadowActivity`, but on `ShadowApplication`.
Do you mind changing it?
```suggestion
                    + " ShadowApplication#setUnbindServiceCallsOnServiceDisconnected(false). This will"
```



@hoisie this warning has been around since at least 2019. Do you know if it is still planned to make `ShadowApplication#setUnbindServiceCallsOnServiceDisconnected(false)` the default behavior?